builtin/commit.c

```diff
@@ -1381,6 +1388,19 @@ int cmd_status(int argc, const char **argv, const char *prefix)
 
 	s.is_initial = get_sha1(s.reference, sha1) ? 1 : 0;
 	s.ignore_submodule_arg = ignore_submodule_arg;
+	s.verbose = verbose;
+	if (s.verbose > 1 && status_format == STATUS_FORMAT_PORCELAIN) {
+		/* Capture extra data for very verbose porcelain output. */
+		s.vvp.enabled = 1;
+		if (!s.is_initial)
+			hashcpy(s.vvp.sha_commit, sha1);
```

Comment:
I'll move the s.verbose = verbose line inside the if() on next line.  This is OK for this code path and I guess preserves the existing mixed usage (where s.verbose is only set in the STATUS_FORMAT_NONE and _LONG cases below).